tests/filters.js

```diff
@@ -86,6 +86,18 @@
             finish(done);
         });
 
+        it('dump', function(done) {
+            equal('{{ {"container":[{\'a\': true}, {\'b\': 2 }]} | dump }}',
```

Comment:
Again I think you want to illustrate dump works on different primitives, so I'd use `['a', 1, { b : false}]` (same as in doc examples) in all tests. The only thing you'd change between test cases is the space parameter (none, 2 spaces, 1 tab).